package.json

```diff
@@ -5,10 +5,33 @@
   "repository": "facebook/relay-starter-kit",
   "version": "0.1.0",
   "scripts": {
-    "start": "babel-node ./server.js",
-    "update-schema": "babel-node ./scripts/updateSchema.js"
+	"start": "./node_modules/babel-cli/bin/babel-node.js ./server.js",
```

Comment:
Did you find this to be necessary? My understanding is that npm executes scripts using the binaries installed via `npm install`. You shouldn't have to explicitly reach into `node_modules`.